### PR TITLE
Guide for WIFI/ESP connection for Original Prusa MINI

### PR DESCRIPTION
**Pull request by zoltan-l**
Corrected and formatted files for ESP Guidelines

### CODE REVIEW

**Comment by foobarjimmy** [comment]
I've compiled latest master branch,flashed ESP using ESP update and it connects to wireless network but freezes when return to menu.

**Comment by zoltan-l** [comment]
Which version type did you compile ? Debug, release, .bbf?, bin?

**Comment by foobarjimmy** [comment]
Release. Firmware was in .bbf

**Comment by zoltan-l** [comment]
> Release. Firmware was in .bbf.It was compiled with “ —preset mini —generate-bbf” and all the other perimeters were not set.

I will check it and let you know, thanks for notification

**Comment by vorner** [comment]
Could you describe few more details what happens? Does the whole printer freeze? Does it unfreeze after a while or is it stuck forever? Which menu, the main settings menu? Does it work after reboot?

File: doc/ESP/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Run **Settings->ESP Update->FLASH ESP AT**
+- Wait until you will see the "**ESP successfully flashed. WiFI initiation started**"  message
+- If you already have loaded your ssid and password into the printer, you should see the WiFi icon ON and in **Settings->Lan settings** you should find the assigned IP address
+- - If no ssid and password is loaded, proceed with next chapter..
+
+
+## Load network data ##
+
+
+- In the prusa`_`printer_settings.ini file and under `[wifi]` tag edit the ssid and psk fields with the correct WiFi data
```
Comment (vorner) [inline doc/ESP/README.md:21]: I was actually suggesting to put the whole file name as code snippet/monospace. Have you tried looking at how github renders it?

File: doc/ESP/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+[`wifi`]
+
+```ssid= enter here the SSID of your wireless network```
+
+```psk= enter here the valid password```
+
+```key_mgmt=WPA```
```
Comment (vorner) [inline doc/ESP/README.md:29]: The tripple ones are for bigger snippets, so it's:

Tripple backtickts
multiple lines of stuff
more lines
tripple backticks

That renders it as a single block of „code“

File: doc/ESP/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+- Store the prusa`_`printer_settings.ini file in the root folder of the USB flash drive and plug in to the printer
```
Comment (vorner) [inline doc/ESP/README.md:31]: Again…

**Review by vorner — CHANGES_REQUESTED** [review]
Again, bits of formatting. I'd provide some suggestions, but the comments here are also markdown and that makes it a bit hard 😇.

**Comment by foobarjimmy** [comment]
The whole printer was frozen and stuck forever after I return to the very top menu(with icons print,filament,settings,etc.).If I didn't go back to the top menu,everything(in the settings) worked just fine.And I had to toggle wifi on and off several times to get it connected but I thought that was something related to wireless network not the firmware.

**Comment by zoltan-l** [comment]
I have followed the guide step ba step, used 4.3.3 +374 locel build by python. The printer did not, stop, so we really need a detailed step by step description.

**Review by vorner — APPROVED** [review]
I think we are good

**Comment by foobarjimmy** [comment]
> I have followed the guide step ba step, used 4.3.3 +374 local. .bbf file build by python. The printer did not, stop!freeze, so we really need a detailed step by step description what you did.

It's stuck on the main menu with usb stick attached.Removing stick when this happen solves the issue.

I connected ESP and usb stick to the printer.Flashed the esp using menu items just like the guide.And loaded settings from usb stick but I didn't remove it after loading settings.Address was shown without problems in the lan settings.But printer froze after I exited to the main menu.It did unfreeze once I remove the usb stick.

**Pull request merged**